### PR TITLE
support change blb name in console

### PR DESCRIPTION
**Pull request by hello2mao**
This pr:
(1)support change blb name in console
(2)add blb id to Service annotation

### CODE REVIEW

File: pkg/cloud-provider/load_balancer.go [hunk 1]
```diff
@@ -25,52 +25,60 @@ import (
...
 	"k8s.io/cloud-provider-baiducloud/pkg/sdk/blb"
 	"k8s.io/cloud-provider-baiducloud/pkg/sdk/eip"
 	"k8s.io/cloud-provider-baiducloud/pkg/sdk/util"
 	"k8s.io/cloud-provider-baiducloud/pkg/sdk/vpc"
 )
 
-const ServiceAnnotationLoadBalancerInternalVPC = "service.beta.kubernetes.io/cce-load-balancer-internal-vpc"
+type AnnotationRequest struct {
+	LoadBalancerId          string
+	LoadBalancerInternalVpc string
```
Comment (m3ngyang) [inline pkg/cloud-provider/load_balancer.go:38]: `LoadBalancerInternalVpc bool`

File: pkg/cloud-provider/load_balancer.go [hunk 2]
```diff
@@ -25,52 +25,60 @@ import (
...
+func (bc *BCECloud) getBCELoadBalancerById(id string) (lb *blb.LoadBalancer, exists bool, err error) {
 	args := blb.DescribeLoadBalancersArgs{
-		LoadBalancerName: name,
+		LoadBalancerId: id,
 	}
 	lbs, err := bc.clientSet.Blb().DescribeLoadBalancers(&args)
 	if err != nil {
-		glog.V(2).Infof("getBCELoadBalancer  not exists blb! %v", args)
-		return blb.LoadBalancer{}, false, err
+		glog.V(2).Infof("getBCELoadBalancer blb not exists: %v", err)
```
Comment (m3ngyang) [inline pkg/cloud-provider/load_balancer.go]: glog.V(2).Infof("getBCELoadBalancer blb %s not exists: %v", args. LoadBalancerId ,err)

File: pkg/cloud-provider/load_balancer.go [hunk 3]
```diff
@@ -79,113 +87,117 @@ func (bc *BCECloud) getBCELoadBalancer(name string) (lb blb.LoadBalancer, exists
 // Parameter 'clusterName' is the name of the cluster as presented to kube-controller-manager
 func (bc *BCECloud) EnsureLoadBalancer(ctx context.Context, clusterName string, service *v1.Service, nodes []*v1.Node) (*v1.LoadBalancerStatus, error) {
 	glog.V(4).Infof("baidubce.EnsureLoadBalancer(%v, %v, %v, %v, %v, %v, %v, %v, %v)",
-		clusterName, service.Namespace, service.Name, bc.Region, service.Spec.LoadBalancerIP, service.Spec.Ports, service.Annotations)
+		clusterName, service.Namespace, service.Name, bc.Region, service.Spec.LoadBalancerIP, service.Spec.Ports,
+		service.Annotations)
+	_, request := ExtractAnnotationRequest(service)
 	if len(service.Spec.Ports) == 0 {
-		return nil, fmt.Errorf("requested load balancer with no ports")
+		return nil, fmt.Errorf("requested load balancer with no ports\n")
```
Comment (m3ngyang) [inline pkg/cloud-provider/load_balancer.go]: return的error不需要换行

File: pkg/cloud-provider/load_balancer.go [hunk 4]
```diff
@@ -25,52 +25,60 @@ import (
...
 	}
 	lbs, err := bc.clientSet.Blb().DescribeLoadBalancers(&args)
 	if err != nil {
-		glog.V(2).Infof("getBCELoadBalancer  not exists blb! %v", args)
-		return blb.LoadBalancer{}, false, err
+		glog.V(2).Infof("getBCELoadBalancer blb not exists: %v", err)
+		return &blb.LoadBalancer{}, false, err
 	}
 
 	if len(lbs) < 1 {
```
Comment (m3ngyang) [inline pkg/cloud-provider/load_balancer.go]: 这里判断 `if len(lbs)!=1` 就好了吧，log 里输出 lbs 列表

**Review by m3ngyang — APPROVED** [review]

File: pkg/cloud-provider/load_balancer.go (hunk 1 above)
Comment (hello2mao) [reply pkg/cloud-provider/load_balancer.go:38]: Not support bool.
See: https://github.com/kubernetes/ingress-nginx/issues/1990

**Pull request merged**